docs/toi.md

```diff
@@ -0,0 +1,41 @@
+
+TOI offers strong data consistency guarantees for online schema changes in PXC. However, it comes with the trade-off of potentially impacting performance and introducing complexities. Weigh the advantages and disadvantages based on your specific needs and the complexity of your schema changes.
+
+## Advantages
+
+| Advantage              | Description                                                                                                                                                                      |
+|------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Strong Data Consistency| TOI makes sure that all nodes in the PXC cluster apply schema changes in the exact same order. This means all nodes have identical schemas at any given time, preventing inconsistencies and potential data corruption. |
+| Simplicity             | You can set the configuration option, `wsrep_OSU_method=TOI`.                                                                                                                     |
+| Predictability         | Since all nodes apply changes in the same order, you can predict the outcome, minimizing surprises during schema updates.                                                         |
+
+## Disadvantages
+
+| Disadvantage            | Description                                                                                                                                                                                                                  |
+|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Blocking Commits        | During a TOI schema change, all other transactions on the cluster are blocked until the schema change is completed on all nodes. This can lead to temporary performance slowdowns or pauses in application activity depending on the complexity of the schema change and workload. |
+| Increased Load          | Replicating the complete schema change statement to all nodes can put additional load on the network and cluster resources.                                                                                                  |
+| Potential Deadlocks     | In rare cases, complex schema changes involving multiple tables or operations might lead to deadlocks. YOu may need to manually resolve these issues.                                                                        |
+| Error Handling          | If the schema change fails on one node, TOI doesn't perform rollback on other nodes. This can leave the cluster in an inconsistent state, requiring additional steps to recover.                                              |
```

Comment:
If the schema change fails on one node, TOI doesn't perform `a` rollback on other nodes.